internals/maintenance_scripts.py

```diff
@@ -557,6 +556,7 @@ def prepare_for_activation(self, stage: Stage, stage_dict: StageDict) -> None:
       self.handle_activation(stage, stage_dict)
     else:
       stage.ot_activation_date = date
+      stage.ot_setup_status = OT_CREATED
```

Comment:
Should we set OT_CREATED as soon as the trial creation succeeds, in the `handle_creation` method?

Reply:
I think that's fine - it will need to be determined how to handle activation, and the ot_setup_status` value could change later down the road, but I've added an additional statement to set this value for clarity